shell/install.sh

```diff
@@ -186,6 +186,22 @@ usage() {
     echo "using '--fresh' or '--dev'"
 }
 
+# Conditionally connect /dev/tty when
+# the environment variable TTY is set to 1
+if ${TTY+false}; then
```

Comment:
isn't this construct `bash` specific? Also don't forget to put quotes around variables, otherwise people can inject random code using the environment variable

Reply:
The `${TTY+false}` syntax passes https://www.shellcheck.net/ and is advertised as a POSIX shell feature [here](https://www.grymoire.com/Unix/Sh.html#uh-33). But now that I read the explanation, I am puzzled why it would work at all. :thinking:

Reply:
I changed the conditional to `TTY="${TTY-0}"` - that is shorter and makes more sense.

Reply:
ah right nevermind sorry, i thought it was something bash added. It is indeed POSIX-complient and present as an example in the man page for POSIX sh(1): https://www.unix.com/man-page/posix/1p/sh/